randovania/games/samus_returns/exporter/patch_data_factory.py

```diff
@@ -275,6 +284,7 @@ def create_data(self) -> dict:
                 "reverse_area8": self.configuration.reverse_area8,
             },
             "text_patches": self._static_text_changes(),
+            "spoiler_log": self._credits_spoiler(),
```

Comment:
If you want to remove the spoiler log for races, you could do 
```python 
"spoiler_log": self._credits_spoiler() if self.description.has_spoiler else {},
``` 
for now.
That would export an empty spoiler log for races but still triggers the patcher to create the headlines (but that's not a big deal)

Reply:
Will do